src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/GCRefMapBuilder.cs

```diff
@@ -120,20 +120,8 @@ public void GetCallRefMap(MethodDesc method, bool isUnboxingStub)
 
             for (uint pos = 0; pos < nStackSlots; pos++)
             {
-                int ofs;
-
-                if (_target.Architecture == TargetArchitecture.X86)
```

Comment:
I realize that we don't yet support Crossgen2 for x86, but is removing this special case correct? Should it instead be moved to the OffsetFromGCRefMapPos function?

Reply:
This special case is already in `OffsetFromGCRefMapPos` overload for x86: https://github.com/dotnet/runtime/blob/6ce761095b57ed38242c8140bc4f1d6da7251567/src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/TransitionBlock.cs#L375-L385